lib/exposes.js

```diff
@@ -554,5 +554,9 @@ module.exports = {
             .withFeature(new Enum('user_type', access.SET, ['unrestricted', 'year_day_schedule', 'week_day_schedule', 'master', 'non_access']).withDescription('Type of user, unrestrictied: owner (default), (year|week)_day_schedule: user has ability to open lock based on specific time period, master: user has ability to both program and operate the door lock, non_access: user is recognized by the lock but does not have the ability to open the lock'))
             .withFeature(new Binary('user_enabled', access.SET, true, false).withDescription('Whether the user is enabled/disabled'))
             .withFeature(new Numeric('pin_code', access.SET).withDescription('Pincode to set, set pincode to null to clear')),
+        squawk: () => new Composite('squawk', 'squawk')
```

Comment:
Since the squawk converter is not in place yet, I cannot review this (please move this to the squawk pr)

Reply:
@Koenkk Added tz.squawk pr here

Reply:
Thanks please also add the `tz.squawk` to the applicable devices and use the exposes there.

Reply:
I will, waiting for you to approve these first :) and when we are sure about the tz.warning, I will do a PR of the updated Climax (now working as expected with the modified tz.warning) and the new Develco device.

Reply:
`tz.squawk` looks good

Reply:
@Koenkk Cool, can you approve all the PR's now?

Reply:
If you add the `tz.squawk` and exposes it is ready for merge.

Reply:
Add where? I will do the PR of both updated Climax and Develco SIRZB-110 when exposes and tz.squawk are merged.  Will also need the tz.warning 'issue' to be resolved. I did even open a post on HA forum asking for more testers. Only one have responded, using the Tuya 0216 Siren. Have no clue if Tuya follows Zigbee specs.

Reply:
> I will do the PR of both updated Climax and Develco SIRZB-110 when exposes and tz.squawk are merged.

I don't see any reason to add it in this PR or am I missing something?